kernel/main.cpp

```diff
@@ -113,11 +113,34 @@ extern "C" void px_kernel_main(const multiboot_info_t* mb_struct, uint32_t mb_ma
     px_rs232_print("\n");
     px_rs232_print(model);
     px_rs232_print("\n");
-    // Now that we're done make a joyful noise
+
+    px_kprintf(DBG_WARN "Mapping in test pages\n");
```

Comment:
Do we want to keep the tests now that we know it works?

Reply:
Maybe we could move it somewhere else